configs/BETAFPVH725/config.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU        STM32H725
+
+#define BOARD_NAME           BETAFPVH725
+#define MANUFACTURER_ID      BEFH
+
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+
+#define BEEPER_PIN           PC9
+
+#define MOTOR1_PIN           PC6
+#define MOTOR2_PIN           PC7
+#define MOTOR3_PIN           PB0
+#define MOTOR4_PIN           PB1
+
+#define LED_STRIP_PIN        PB10
+
+#define UART1_TX_PIN         PA9
+#define UART1_RX_PIN         PA10
+
+#define UART2_TX_PIN         PA2
+#define UART2_RX_PIN         PA3
+
+#define UART3_TX_PIN         PC10
+#define UART3_RX_PIN         PC11
+
+#define UART4_TX_PIN         PA0
+#define UART4_RX_PIN         PA1
+
+#define UART5_TX_PIN         PC12
+#define UART5_RX_PIN         PD2
+
```

Comment:
`#define MSP_DISPLAYPORT_UART SERIAL_PORT_UART5`

Reply:
Done.

Reply:
Not seeing this change. Are you sure you've pushed your commits?

Reply:
Sorted now.

Reply:
@SteveCEvans should be sorted now